### DISCUSSION TITLE
Can't set content type to instance

### DISCUSSION THREAD

**Discussion by DZ8540** [axios/axios#4592] [Q&A] [answered]
Hello there! I sended query via axios instance with several headers

![telegram-cloud-photo-size-2-5366168114724715501-x](https://user-images.githubusercontent.com/50152205/162405280-d3897635-b5ba-4248-b04a-7ac24b8a71e6.jpg)

But query do not set content type header (see request headers in screenshot below)

![telegram-cloud-photo-size-2-5366168114724715500-x](https://user-images.githubusercontent.com/50152205/162405492-97475a22-4290-4c6b-a7ec-fde3d595fabe.jpg)

What could be the problem???

**Comment by DZ8540** [comment, marked as answer]
In 0.26.1 axios version content type header work, but i need use the 0.21.4 axios version!

**Comment by gikami** [comment]
I have the same error. Please fix the version.
